compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java

```diff
@@ -2467,7 +2479,9 @@ void addCompoundAssignmentStatement(DiagnosticPos pos, Set<Whitespace> ws, Strin
         BLangCompoundAssignment assignmentNode =
                 (BLangCompoundAssignment) TreeBuilder.createCompoundAssignmentNode();
         assignmentNode.setExpression(exprNodeStack.pop());
-        assignmentNode.setVariable((BLangVariableReference) exprNodeStack.pop());
+
+        BLangVariableReference varref = (BLangVariableReference) exprNodeStack.pop();
```

Comment:
Do we need this change?

Reply:
This was a left over from compound assignment validation, will remove this.